src/AppInstallerCLICore/Commands/SourceCommand.h

```diff
@@ -37,7 +37,7 @@ namespace AppInstaller::CLI
 
     struct SourceListCommand final : public Command
     {
-        SourceListCommand(std::string_view parent) : Command("list", parent) {}
+        SourceListCommand(std::string_view parent) : Command("list", { "ls" }, parent) {}

```

Comment:
This came to mind while I was adding the alias for the `pin` sub-commands. I don't know if we want to make this a separate PR so it goes out sooner than pinning.

Reply:
I would suggest a separate PR just to keep the git history cleaner

Reply:
Yeah, you're right... I have the bad habit of seeing things that I want to change but then being too lazy to do it in a separate PR...

Anyways, here it is #2736